test/modules/string/replace_test.dart

```diff
@@ -0,0 +1,33 @@
+import 'package:flodash/flodash.dart' as flodash;
+
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group('replace', () {
```

Comment:
This group isn't needed ne, all of the tests inside this file will be related to the replace function only. It's adding a little bit of additional nesting, what do you think ?

Reply:
yes i will remove it